homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -68,17 +70,19 @@
     """Set up the Frontier Silicon platform."""
     if discovery_info is not None:
         async_add_entities(
-            [AFSAPIDevice(discovery_info["ssdp_description"], DEFAULT_PASSWORD)], True
+            [AFSAPIDevice(discovery_info["ssdp_description"], DEFAULT_PASSWORD, "")],
```

Comment:
Please use the default empty value `None`.
```suggestion
            [AFSAPIDevice(discovery_info["ssdp_description"], DEFAULT_PASSWORD, None)],
```

Reply:
Great, that's neater. Sorry I'd misinterpreted the docs and thought that a non-null default was required for optional parameters.